src/components/molecules/OcTable/OcTable.spec.js

```diff
@@ -195,15 +195,30 @@ describe("OcTable", () => {
     expect(wrapper.findAll(".oc-table-highlighted").length).toEqual(2)
   })
 
-  it("adds data-file-id for rows", () => {
+  it("adds data-item-id for rows", () => {
     const wrapper = shallowMount(Table, {
       propsData: {
         fields,
         data,
         highlighted: [],
       },
     })
-    expect(wrapper.html().indexOf("data-file-id")).toBeGreaterThan(-1)
+    console.log(wrapper.html())
```

Comment:
dev leftover?

Reply:
Ah yeah I added this in my review for the original PR by @fschade, not relevant in the context of this one as it is only supposed to run against the web CI for a quick check